content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,138 @@
+
+```python
+marker.id in WALL  # Is this marker a wall marker
+```
+
+{{% notice tip %}}
+It's recommended to use `marker.is_wall_marker()` instead of `marker.id in WALL`
+{{% /notice %}}
+
+
+### _Column_ Markers
+These markers are those positioned on the 4 faces of the 4 columns in the arena. Each face has a unique, documented ID, and so could be used in addition to [_Wall_ Markers](#wall-markers) to calculate your position, and make sure you don't run into them!
+
+```python
+marker.id in COLUMN
+>>> True
+```
+
+Each column has its own constant to check if the marker belongs to a specific column. There's also additional constants to check if a marker is facing a certain direction, and of course checking a specific face of a specific column.
+
+{{% notice tip %}}
+Columns are referenced like the points of a compass (_N_, _E_, _S_, _W_), with the North West corner of the arena as corner 0.
+{{% /notice %}}
+
+```python
+marker.id in COLUMN  # Is the marker a column marker?
+ 
+marker.id in COLUMN_N  # Is the marker on the North column? 
+
+marker.id in COLUMN_FACING_E  # Is the marker facing East on its column?
+ 
+marker.id == COLUMN_S_FACING_W  # Is the marker on the South column facing West?
+```
+
+In the above examples, the points of the compass may be replaced with any others. 
+
+{{% notice note %}}
+The constants for the general classes of column markers (e.g: `COLUMN_N`) are collections of marker ids and so need comparison with `in`. The constants for the specific column faces (e.g: `COLUMN _S_FACING_W`) define only a single value and so should be compared to using `==`.
+{{% /notice %}}
+
+### _Token_ Markers
+Each token has a marker on its face. These markers can be used to work out what teams token it is.
+
+Each scoring zone has a corresponding constant that matches tokens belonging to that zone. There's also another constant that matches tokens belonging to any zone.
+
+```python
+marker.id in TOKEN  # Is the marker on a token?
+
+marker.id in TOKEN_ZONE_0  # Does this token belong to the team in zone 0?
+```
+
+{{% notice info %}}
+Each face of the token has the same marker on. However each token has different markers.
+{{% /notice %}}
+
+{{% notice tip %}}
+It's recommended to use `marker.is_token_marker()` instead of `marker.id in TOKEN`.
+{{% /notice %}}
+
+### All Marker Constants
+Any of the below marker constants can be imported from the `robot` module.
+
+#### Markers on a specific column
+A set of marker ids on any face of a specific column.
+
+- `COLUMN_N`
+- `COLUMN_E`
+- `COLUMN_S`
+- `COLUMN_W`
+
+#### Markers facing a specific direction
+A set of marker ids specifying which direction a column is facing.
+
+- `COLUMN_FACING_N`
+- `COLUMN_FACING_E`
+- `COLUMN_FACING_S`
+- `COLUMN_FACING_W`
+ 
+#### Specific column faces
+A single marker ids specifying which column the marker is on, and which direction it's facing. 
```

Comment:
"ids" -> "id" (or even better, "ID")

Reply:
Not sure i'll ever use id/ID correctly, however it's meant to be? (cc @PeterJCLaw )

Agreed it should be singular though

Reply:
I think lowercase is probably right (it's not an acronym), but I don't feel strongly. Agree it should be singular.

Reply:
Hrm, in some scenarios it could be an acronym for "Identity Document", though I think here it's just a contraction of "identity".

Reply:
According to [the internet](https://stackoverflow.com/questions/1151338/id-or-id-on-user-interface), _ID_ is fine, however it should behave as _Id_ in camel-case environments. Personally, i'm ok with this! Although changing it might be out of scope for this PR